docs/changelog.rst

```diff
@@ -3,4 +3,4 @@
 Changelog
 =========
 
-.. include:: ../CHANGELOG.rst
+.. include:: ../CHANGELOG.md
```

Comment:
What would then be best approach? Convert `CHANGELOG.md` to rst and put it in `docs/changelog.rst`, Convert `CHANGELOG.md` to `CHANGELOG.rst` in root dir and leave `docs/changelog.rst` as it is or something else?